lib/rubocop/config.rb

```diff
@@ -181,7 +181,8 @@ def path_relative_to_config(path)
     # directory since that wouldn't work.
     def base_dir_for_path_parameters
       @base_dir_for_path_parameters ||=
-        if File.basename(loaded_path).start_with?('.rubocop') &&
+        if loaded_path &&
```

Comment:
This change seems unrelated.

Reply:
Some explanation here would be useful.

Reply:
It seems to be necessary to get the build to pass. I don't understand why - I agree it seems unrelated. I suspect it might be related to the use of `minimum_target_ruby_version`, which seems infrequently used?

Re the build - I'm confused about the current set of CI failures, which I'm unable to reproduce locally. Any suggestions about where to start looking?

Reply:
This becomes unnecessary and the build becomes happy without `minimum_target_ruby_version`. :/

Reply:
This code should not be updated as it is a change unrelated to the addition of a new cop.

Reply:
> Re the build - I'm confused about the current set of CI failures, which I'm unable to reproduce locally. Any suggestions about where to start looking?

Could you try the following?
https://github.com/rubocop-hq/rubocop/pull/7663/files#r371044684